conjur/util/__init__.py

```diff
@@ -0,0 +1,5 @@
+"""
+util
+
+This package is for util modules
```

Comment:
utils module or package?

Reply:
Each file in the package is a util module. The folder is a package

Reply:
did you use the existing Util folder or create a new one? a util folder already exists

Reply:
https://github.com/cyberark/conjur-api-python3/tree/refactor-directories/Utils Maybe this should be moved into Conjur

Reply:
Unified util folders